skimage/measure/fit.py

```diff
@@ -156,6 +169,151 @@ def predict_y(self, x, params=None):
         return (dist - x * math.cos(theta)) / math.sin(theta)
 
 
+class LineModelND(BaseModel):
+    """Total least squares estimator for N-dimensional lines.
+
+    Lines are defined by a point (origin) and a unit vector (direction).
+
+    Attributes
+    ----------
+    params : tuple
+        Line model parameters in the following order `origin`, `direction`.
+        These parameters correspond to the vector equation
```

Comment:
GitHub autoformatting is unhelpful in this case. The necessary tweaks for Sphinx:

* Backticks should remain around "origin" and "direction"
* You need **two** colon symbols after 'equation'
* Indent the equation by two spaces relative to the prior text block
* Do not put the new equation in backticks.